test/e2e/apps/types.go

```diff
@@ -29,7 +29,7 @@ const (
 
 var (
 	// CronJobGroupVersionResourceBeta unambiguously identifies a resource of cronjob with beta status
-	CronJobGroupVersionResourceBeta = schema.GroupVersionResource{Group: "batch", Version: "v1beta1", Resource: "cronjobs"}
+	CronJobGroupVersionResourceBeta = schema.GroupVersionResource{Group: "batch", Version: "v1", Resource: "cronjobs"}
```

Comment:
Here, similarly to above, drop entirely this variable and all related skip checks.